Code/Java/greedyalgo.java

```diff
@@ -0,0 +1,56 @@
+/*
+Calulating Min use of Coin or Notes for Arranging Specific Amount
+Logic is to take coins with greater value first. This can reduce
+the total number of coins needed. Start from the largest possible denomination and
+then keep adding denominations while the remaining value is greater than 0.
+after all the size of ans containing notes / coins were used for min notes is
+Final Answer or Number of Minimum Notes for Arranging a fix value
+*/
+
+package algo;
+import java.util.Scanner;
+
+public class algojava {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        // After Demonitization in India Due to Launch of 200 and 2000 Notes in Market :)
+        int[] arr = {1,2,5,10,20,50,100,200,500,1000,2000};
```

Comment:
just give spaces after "," it would be more readable and a good practice to follow